ecs-cli/modules/cli/compose/entity/task/task_test.go

```diff
@@ -386,3 +387,101 @@ func TestBuildRuntaskInputManagedTagsDisabled(t *testing.T) {
 		assert.Nil(t, req.EnableECSManagedTags, "Expected ECS Managed tags to be unset")
 	}
 }
+
+func TestBuildRunTaskInput_EFSFargate(t *testing.T) {
+	taskDef := "dogPicService"
+	count := 1
+	cluster := "myCluster"
+	launchType := config.LaunchTypeFargate
+	flagSet := flag.NewFlagSet("ecs-cli", 0)
+	flagSet.Bool(flags.DisableECSManagedTagsFlag, true, "")
+	cliContext := cli.NewContext(nil, flagSet, nil)
+	ctrl := gomock.NewController(t)
+	mockEcs := mock_ecs.NewMockECSClient(ctrl)
+	context := &context.ECSContext{
+		ECSClient:  mockEcs,
+		CLIContext: cliContext,
+		ECSParams:  ecsParamsWithEFSVolume(),
+		CommandConfig: &config.CommandConfig{
+			Cluster:    cluster,
+			LaunchType: launchType,
+		},
+	}
+
+	task := &Task{
+		ecsContext: context,
```

Comment:
Given that the test assertion only checks the name of the task def (and nothing about the actual EFS data), do you think it'd be worth asserting that part is all good or just assume the unit tests on convert_task_definition.go are enough?

Reply:
This test checks the platform version in the RunTaskInput struct, which I think is sufficient here.